lib/Components/AbstractComponent.js

```diff
@@ -9,13 +9,7 @@ export class AbstractComponent extends Events {
         super();
         
         this.components = [];
-        this.options = options;
-
-        //
-        // if (module.hot) {
-        //     this.constructor.instances = this.constructor.instances || [];
-        //     this.constructor.instances.push(this);
-        // }
+        this.options = options || {};
```

Comment:
We can use defaults notation in arguments `options = {}` and then just assign options